pages/common/scp.md

```diff
@@ -1,30 +1,28 @@
 # scp
 
-> Copies files between hosts on a network.
-> Works over a secure connection (SSH).
+> Secure copy.
+> Copy files between hosts using Secure Copy Protocol over SSH.
 
-- Upload a file, or upload and rename a file:
+- Copy a loocal file to a remote host:
 
-`scp {{/local/file.txt}} {{10.0.0.1}}:{{/remote/path/}}`
-`scp {{/local/file.txt}} {{10.0.0.1}}:{{/remote/path/newname.txt}}`
+`scp {{local_file}} {{remote_host}}:{{/path/remote_file}}`
 
-- Download a file:
+- Copy a file from a remote host to a local folder:
 
-`scp {{10.0.0.1}}:{{/remote/path/file.txt}} {{/local/folder}}`
+`scp {{remote_host}}:{{/path/remote_file}} {{/path/local_dir}}`
```

Comment:
That was actually intentional.  I wanted to offer the three styles of copy (file->file, file->dir, recursive dir->dir) in only three examples.  But if you think "local file to remote file", "remote file to local file" and "dir to dir" are better without any mention that the filename is optional on the right side, I'd be fine with that too.